sass/_content.scss

```diff
@@ -5,287 +5,319 @@
 10.1 Posts and pages
 --------------------------------------------------------------*/
 .sticky {
-    background: #333;
+  background: $color-dark-grey-alt;
 
-    .entry-title a {
-        color: #fff;
-        padding-bottom: 2px;
+  .entry-title a,
+  .entry-meta a,
+  .entry-summary a {
+    color: $color-white;
+    padding-bottom: 2px;
 
-        &:hover {
-            border-color: #fff;
-            color: #fff;
-        }
+    &:hover {
+      border-color: transparent;
+      color: $color-white;
     }
+  }
 
-    .entry-subtitle a {
-        color: #aaa;
+  .entry-title a:hover {
+    border-color: $color-white;
+  }
 
-        &:hover {
-            border-color: #999;
-            color: #aaa;
-        }
-    }
+  .entry-meta a,
+  .entry-summary a {
+    border-color: $color-white;
 
-    * {
-        color: #fff;
+    &:hover {
+      border-color: transparent;
     }
+  }
 
-    &:after {
-        display: none;
+  .entry-subtitle a {
+    color: #aaa;
+
+    &:hover {
+      border-color: #999;
+      color: #aaa;
     }
-}
+  }
 
-.hentry {
-	clear: both;
-	margin: 0;
-    padding: 50px 0;
-    position: relative;
+  a.author {
+    border-color: transparent;
 
-    &:after {
-        background: rgba(0, 0, 0, .15);
-        bottom: 0;
-        content: '';
-        height: 1px;
-        left: 0;
-        position: absolute;
-        right: 0;
-        @include site-width;
+    .name {
+      border-color: $color-white;
     }
 
-    .single &:after {
-        display: none;
+    &:hover {
+      .name {
+        border-color: transparent;
+      }
     }
+  }
 
-	.single &,
-	.page & {
-        border: 0;
-        padding: 0;
-    }
+  * {
+    color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.